### DISCUSSION TITLE
FBW

### DISCUSSION THREAD

**Discussion by FreekMulder** [flybywiresim/aircraft#6700] [Help]
Hello Simmers, 
After each time I start my computer, FBW ask me to update patch version 0.7.4.
Anoying!!
Please help me to find my "blindspot"
Happy landings, Freek (PH-FGM)

**Comment by ExampleWasTaken** [comment]
Try to delete the `flybywire-aircraft-a320-neo` folder from your community folder and download the aircraft through the installer again.
